dashboard/test/ui/features/teacher_tools/rubrics/ai_evaluate_student_code.feature

```diff
@@ -150,6 +152,7 @@ Feature: Evaluate student code against rubrics using AI
     # Teacher switches to Class Management tab
     When I click selector "button:contains('Class Data')"
     And I wait until element ".uitest-run-ai-assessment-all" is visible
+    And I wait for 2 seconds
```

Comment:
sorry for not commenting on this one earlier, but can we get rid of this wait too? I think the next line can just be done like this: https://github.com/code-dot-org/code-dot-org/blob/769ab665265dcddf3ac395e2ea4af340ba033d51/dashboard/test/ui/features/teacher_tools/rubrics/teacher_view_of_rubric.feature#L35

Reply:
Oof, I thought I got all of them